lib/Bio/EnsEMBL/DataCheck/Checks/MetaKeyAssembly.pm

```diff
@@ -53,7 +53,7 @@ sub tests {
       cs.attrib RLIKE 'default_version' AND
       at.code = 'toplevel' AND
       m.meta_key = 'assembly.default' AND
-      cs.version <> m.meta_value AND
+      MD5(cs.version) <> MD5(m.meta_value) AND
```

Comment:
Just a question, how big are the table on which you want to compute this? Because applying any function imply a MySQL full table-scan.

Reply:
I tested it quickly on mentioned db, and moving to binary produce a gain of ~30%, not sure this is very important.

Reply:
Hi @marcoooo , the meta and coord_system tables are small in my experience. But if you or @james-monkeyshines know of cheaper ways of comparing two strings with case-sensitivity I can try them, sure.

Reply:
I was just thinking of using BINARY comparison instead of MD5. I actually don't know which one is faster. the only thing is that BINARY is a KEYWORD, and MD5 a mysql function. There stops my knowledge of both :-)

Reply:
I will try BINARY, it seems like a cast. That should be faster than a function, but I need to make sure it works as expected

Reply:
Quick test on pl1 with musa_acuminata_core_51_104_2:
i) MD5      39.4 ms
ii) BINARY 79.7 ms
Both seems to do the job